api/src/main/resources/workbench-api.yaml

```diff
@@ -802,6 +806,126 @@ paths:
           description: Internal Error
           schema:
             "$ref": "#/definitions/ErrorResponse"
+
+  "/v1/apps/{workspaceNamespace}":
+    get:
+      summary: Get the user's app by type.
+      description: "Returns the current user's app by type."
+      operationId: getApp
+      tags:
+        - app
+      parameters:
+        - "$ref": "#/parameters/workspaceNamespace"
+      responses:
+        200:
+          description: The app for this user and workspace.
+          schema:
+            "$ref": "#/definitions/App"
+        404:
+          description: No app exists for this user and workspace.
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+        412:
+          description: Compute is temporarily suspended for security reasons.
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+    post:
+      summary: Create a workspace APP.
+      description: |
+        Creates a new app for the current user in the given workspace. If a app already exists for the user in this billing project, a 409 conflict error is returned (even if the app is still initializing or is not in a ready state).
```

Comment:
In Leo API, you changed the description to "App creation request has been received" which implies that the creation is an async process. I think the workbench API description and response should also indicate that the request was received.